code/b-parasite/src/main.c

```diff
@@ -62,8 +62,13 @@ static void rtc_callback() {
   prst_adc_soil_moisture_t soil_read = prst_adc_soil_read(batt_read.voltage);
   prst_pwm_stop();
   nrf_gpio_pin_clear(PRST_FAST_DISCH_PIN);
+
+  nrf_gpio_pin_set(PRST_PHOTO_V);                                         // set GPIO for photoresistor HIGH
+  prst_adc_photo_sensor prst_adc_photo_read(double battery_voltage);      // read PHOTO SENSOR value

```

Comment:
I don't think this line is correct. I believe you're declaring a function here, instead of calling the one you defined. I think something similar to the lines above should work:

```C
prst_adc_photo_sensor_t photo_red =  prst_adc_photo_read(battery_voltage);
```

Reply:
Right... Silly bug. Copied the definition line but didn't change the syntax...
Done.